README.md

```diff
@@ -67,36 +67,34 @@ ide.json.path=subfolder/from/root
 file.json.name=newFileName.json
 ```
 
-**Step 1:** Install `RegisterCompanion` onto the device where you want to mock In-app Billing. You can find the latest version in the [Releases Tab](https://github.com/nytm/Register/releases).
-
-**Step 2:** Add Register as a dependency to your client app:
+**Step 1:** Add Register as a dependency to your client app, this will install the companion app:
 ```groovy 
-compile 'com.nytimes.android:register:0.0.3'
+compile 'com.nytimes.android:register:0.0.8'
 ```
 
 or, on Android Gradle Plugin 3.0 or later:
 
 ```groovy 
-implementation 'com.nytimes.android:register:0.0.3'
+implementation 'com.nytimes.android:register:0.0.8'
 ```
 
-**Step 3:** Create a test Google Services provider (or a real provider):
+**Step 2:** Create a test Google Billing Client (or a real client):
```

Comment:
Our internal implementation is still `GoogleServiceProvider` not a `Billing Client` in this case

Reply:
Ya, I saw that. Although it does look like what users really want to do is create a test implementation of the Billing Client? I could be incorrect but I thought having something less specific to our implementation made more sense for external docs?

Reply:
@goldstei can you chime in here? I think `BillingClientTesting` isn't something that should be visible given its placement in the library

Reply:
@candidcold since user has choice to create a real `BillingClient` or ours, what other option is there?  BTW, I never like `BillingClientTesting`.  Maybe `BillingClientWithRegister` or somesuch?

Reply:
In both the sample and our usages internally we use the `GoogleServiceProvider`'s builder and that manages which implementation of client that we're using. I'm not totally sure if it's a good or a bad thing, it could be a holdover from before we were on Play Billing, but I was just a little wary of recommending something in our readme that isn't something we do in our sample or the way that we use things ourselves.